DESCRIPTION

```diff
@@ -1,13 +1,13 @@
 Package: gpdd
-Title: What the Package Does (One Line, Title Case)
+Title: Download datasets from the Global Population Dynamics Database.
 Version: 0.0.0.9000
 Authors@R: 
```

Comment:
Please update `Authors@R` entry to include and yourself & me.  We can ping @bjb568 & @amy-tong to add themselves as well.

Reply:
This is done.